oeps/processes/oep-0021-proc-deprecation.rst

```diff
@@ -195,6 +195,20 @@ considers the timing of the next `Open edX named release`_.
   Consider when the next Named Release is cut; if it is very soon, you may wish to delay final
   removal until after the cut date.
 
+Consider choosing deprecation and removal dates that allow for a full
+release cycle for transition planning. For example, a deprecation
+proposal could be accepted while Maple is being finalized, then
+implement the removal some time after Maple is released (so that the
```

Comment:
Nit: The removal can actually come as soon as Maple is cut - at that point, all changes will be in Nutmeg.

Reply:
This is true, but I'm a little concerned that removals that happen shortly after branch cut may interfere with fixes that need to be backported. Maybe that's too rare to be concerned about, though.

Reply:
Yeah. I don't feel entirely strongly either way. I think sometimes it can be really annoying to have to hold up artificially another two months, and practically, people are merging breaking things after a given release is cut. This also has the benefit of more time on master and thus more tested before the next release. But! This is why I said "nit" - I would defer to you.

Reply:
Maybe a compromise could be something like,

```
implement the removal some time after Maple is released (so that the
removal itself will land in Nutmeg). Note that while removals technically
can come as soon as a named release is _cut_, it is discouraged as the
removal may interfere with fixes that need to be backported. Any deployment...
```

Reply:
I updated the language a bit -- more or less what you said. I think it's ready for merge if you do. :-) (Would you be acting as Arbiter here? I'm not totally clear on the OEP-update process, even after having looked at OEP-1.)

Reply:
I find OEP-1 written like OEPs will hardly ever change because a change implies a new OEP.

My personal interpretation is a reasonably-sized change to an OEP should have an arbiter, but smaller things (ie, clarifying points or updating the OEP to reflect how we actually operate) don't need one - just good review.

To me this seems like it falls in the latter bucket - it's a small change that mostly just clarifies how we commonly operate. If you think it's larger, the "correct" way to do things would be to first ask the original arbiter and if s/he isn't available, find a new one.

Reply:
Makes sense!